### PR TITLE
Exceptions

### PR DESCRIPTION
**Pull request by Eugenio36**

### CODE REVIEW

File: src/main/java/io/codelex/exceptions/practice/Exercise2.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+public class Exercise2 {
+    public static void main(String[] args) {
+        methodA();
+    }
+
+
+    public static void methodA() throws ArithmeticException {
+        try {
+            methodB();
+        } catch (ArithmeticException e) {
```
Comment (NilsKula) [inline src/main/java/io/codelex/exceptions/practice/Exercise2.java:12]: `After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller`

File: src/main/java/io/codelex/exceptions/practice/Exercise4.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package io.codelex.exceptions.practice;
+
+class NonPositiveNumberException extends Exception {
```
Comment (NilsKula) [inline src/main/java/io/codelex/exceptions/practice/Exercise4.java:3]: kā nested class vai arī atsevišķā failā

**Review by NilsKula — CHANGES_REQUESTED** [review]

File: src/main/java/io/codelex/exceptions/practice/Exercise4.java (hunk 2 above)
Comment (Eugenio36) [reply src/main/java/io/codelex/exceptions/practice/Exercise4.java]: Salabots.

File: src/main/java/io/codelex/exceptions/practice/Exercise2.java (hunk 1 above)
Comment (Eugenio36) [reply src/main/java/io/codelex/exceptions/practice/Exercise2.java:11]: Salabots.

**Review by NilsKula — APPROVED** [review]

**Pull request merged**